src/GAudioOutput.cc

```diff
@@ -56,16 +56,14 @@ GAudioOutput::GAudioOutput(QObject *parent) :
     worker->moveToThread(thread);
     connect(this, &GAudioOutput::textToSpeak, worker, &QGCAudioWorker::say);
     connect(this, &GAudioOutput::beepOnce, worker, &QGCAudioWorker::beep);
+    connect(this, SIGNAL(finished()), thread, SLOT(quit())); //tried terminate() also
```

Comment:
Where is this finished signal coming from on GAudioOutput? Also please don't use the SIGNAL/SLOT syntax. Use the direct reference syntax like the line above this one. SIGNAL/SLOT syntax allows you to connect to signals which don't exist.